infrastructure/terraform/accounts/modules/role_policies/publisher/policies.tf

```diff
@@ -0,0 +1,38 @@
+data "aws_iam_policy_document" "ecr_power_user" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "ecr:GetAuthorizationToken",
+      "ecr:BatchCheckLayerAvailability",
+      "ecr:GetDownloadUrlForLayer",
+      "ecr:GetRepositoryPolicy",
+      "ecr:DescribeRepositories",
+      "ecr:ListImages",
+      "ecr:DescribeImages",
+      "ecr:BatchGetImage",
+      "ecr:InitiateLayerUpload",
+      "ecr:UploadLayerPart",
+      "ecr:CompleteLayerUpload",
+      "ecr:PutImage",
```

Comment:
tiny nit: since there are lots of things here, maybe sort alphabetically so it's easier to see at a glance if a particular permission is included?